src/hooks/useReleaseHistory.tsx

```diff
@@ -40,52 +51,34 @@ export function useReleaseHistory(): ReleaseData[] {
   return releaseHistory;
 }
 
-export function getStaticReleaseData(): NodeReleaseData[] {
-  return [
-    {
-      release: 'v10',
-      status: 'Maintenance LTS',
-      codename: 'Dubnium',
-      initialRelease: '2018-04-24',
-      activeLTSStart: '2018-10-30',
-      maintenanceLTSStart: '2020-05-19',
-      endOfLife: '2021-04-30',
-    },
-    {
-      release: 'v12',
-      status: 'Active LTS',
-      codename: 'Erbium',
-      initialRelease: '2019-04-23',
-      activeLTSStart: '2019-10-21',
-      maintenanceLTSStart: '2020-10-20',
-      endOfLife: '2022-04-30',
-    },
-    {
-      release: 'v14',
-      status: 'Active LTS',
-      codename: 'Fermium',
-      initialRelease: '2020-04-21',
-      activeLTSStart: '2020-10-27',
-      maintenanceLTSStart: '2021-10-19',
-      endOfLife: '2023-04-30',
-    },
-    {
-      release: 'v15',
-      status: 'Curent',
-      codename: '',
-      initialRelease: '2020-10-20',
-      activeLTSStart: '',
-      maintenanceLTSStart: '2021-04-01',
-      endOfLife: '2021-06-01',
-    },
-    {
-      release: 'v16',
-      status: 'Pending',
-      codename: '',
-      initialRelease: '2021-04-20',
-      activeLTSStart: '2021-10-26',
-      maintenanceLTSStart: '2022-10-18',
-      endOfLife: '2024-04-30',
-    },
-  ];
+export function useReleaseData(): NodeReleaseData[] {
```

Comment:
Maybe we can consider caching of this response at least in session storage. So it won't do same request on `/releases` and `/download` pages?

Reply:
Actually I'm raising an issue in next PR to fetch this data during build time rather than run time, as this data is not getting rendered on javascript-disabled browsers as of now. That should fix this